README.md

```diff
@@ -129,11 +129,64 @@ or with Docker
 docker compose up --build
 ```
 
+## How to use it
+
 ### Endpoints documentation
 For endpoints documentation you can visit swagger which is exposed on port 8080 by default.
 ```
 http://localhost:8080/swagger/index.html
 ```
 
+### Authentication
+
+#### Enabling
+
+To setup and enable authentication with token setup the following environment variables
+- HTTP_SERVER_USEAUTH=true
+- HTTP_SERVER_AUTHTOKEN=replace_me_with_token_you_want_to_use_as_admin_token
+
+ℹ️ actually authorization is enabled by default so only the HTTP_SERVER_AUTHTOKEN env is required
```

Comment:
I would update this in line with what is said in it - seems like this was written in hindsight.

Above do not include `HTTP_SERVER_USEAUTH=true` there's then no need to mention it. The HTTP_SERVER_USEAUTH=false below is sufficient to explain.